source/MoorDyn2.hpp

```diff
@@ -630,6 +637,9 @@ class MoorDyn final : public io::IO
 	/// (if using env.WaveKin=1)
 	unsigned int npW;
 
+	/// Disabled writing to output files or console when running
+	bool disableOutput = false;
```

Comment:
We should add this to the input file options list and the docs. I can open a PR to your branch @AlexWKinley.

Reply:
PR is open and ready to merge. It adds this, and also cleans up the options list handling a bit

Reply:
Merged. Thanks for cleaning that up.